### DISCUSSION TITLE
Help with "Cannot find overridden methods" W00015

### DISCUSSION THREAD

**Discussion by schwindyboo** [babel/babel#14212] [Q&A] [answered]
Hi, 

I have implemented an interface from Autofac `Autofac.Core.Activators.Reflection.IConstructorFinder` in my project but am seeing the warnings below when obfuscating

I have tired adding a rule to skip the implementation but I still get this warning

Can anyone help with a rule or option for this issue please?

> BABEL : warning [W00013]: Could not resolve assembly 'Autofac, Version=5.2.0.0, Culture=neutral, PublicKeyToken=17863af14b0044da'. [C:\Dev\MyCompany.AutoFac.csproj]
> BABEL : warning [W00015]: Cannot find overridden methods of 'MyCompany.AutoFac.AllConstructorFinder::FindConstructors(System.Type):System.Reflection.ConstructorInfo[]': Could not resolve assembly: 'Autofac, Version=5.2.0.0, Culture=neutral, PublicKeyToken=17863af14b0044da' [C:\Dev\....csproj]

**Comment by nicolo-ribaudo** [comment, marked as answer]
That error looks like it's not coming from _this_ Babel. Maybe it's https://www.babelfor.net/?

**Reply by schwindyboo** [reply]
Thanks Nicolo 🤦‍♂️
